### ISSUE TITLE
Ctrl-K Ctrl-K is a problem

### ISSUE THREAD

**Issue by peterjcarr** [inouetakuya/obsidian-kill-and-yank#15]
After years and years ... and years of using emacs I am habituated to using Ctrl-K multiple times to select several lines of text. And then Ctrl-Y to paste them. When I do this in current version of Obsidian with the obsidian-kill-and-yank plugin I lose all of the content of the 'kill ring'.

For example,
```
Line one
Line two
Line three
```

Position the cursor at beginning of Line one.
This works: `Ctrl-K` followed by `Ctrl-Y`
This does not: `Ctrl-K` `Ctrl-K` `Ctrl-y`

Workaround: Mark region works. It's just hard to untrain my fingers.

I'm running Version 1.2 of the plugin in Obsidian Version 1.3.4 (Installer 1.2.7) on macOS Ventura (13.3.1 (a) (22E772610a))

**Comment by inouetakuya** [comment]
Hi, Thanks for giving me feedback.

Yes, I understand very much how you feel 😭 

I have thought about the multiple kill spec before.

I made the decision not to support multiple kills to keep things simple.

I also decided not to provide an option to support multiple kills.

Maybe in 3 months I will have a different idea, but that is the way it is now.

Sorry, but I ask that you fork this plugin and support multiple kills yourself.

I am so sorry 🙏 🙏 🙏
